Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:alpine
+
+RUN apk add libxml2-dev && apk add libxslt-dev && apk add build-base && apk add git
+
+WORKDIR /usr/src/app
+
+RUN git clone https://github.com/zaubererty/sc2mqtt.git .
```

Comment:
Please refer to the original repo here, not to a fork.

Reply:
Valid point, going to change that